### DISCUSSION TITLE
Improves of the diagrams backdrop zoom

### DISCUSSION THREAD

**Discussion by Wils3B** [ory/kratos#1936] [Show and tell]
Hello everyone, I am a newbie on ory kratos.
While reading the docs I realized that I could not read the diagrams well when I opened them in zoom

I made some changes locally and I would like to contribute

### Before
![image](https://user-images.githubusercontent.com/44102335/140762667-0b1c592e-b6d3-4807-8a25-68787d317879.png)

### After
![image](https://user-images.githubusercontent.com/44102335/140763068-09e26e7d-e2fc-4383-8fcd-f711bca760ed.png)
![image](https://user-images.githubusercontent.com/44102335/140763219-d8e69303-9130-49b4-9aef-b8697181a81b.png)

Now the image scrolls and the background has no margin at the bottom.

**Comment by vinckr** [comment]
Thanks for looking into this!
You are totally right, some of the diagrams are cut off at the bottom third basically. 
Feel free to open a PR, your fix looks good from the screenshots.
Thanks again @Wils3B 🎉

**Reply by aeneasr** [reply]
Nice! :) Please make the fix over at: https://github.com/ory/docusaurus-template :)

**Reply by Wils3B** [reply]
I sent a pull request at [https://github.com/ory/docusaurus-template/pull/83](https://github.com/ory/docusaurus-template/pull/83), Thanks to make a review
